roles/kubernetes/node/tasks/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 - include: install.yml
 
+- include: nginx-proxy.yml
+  when: is_kube_master == false and loadbalancer_apiserver_localhost
```

Comment:
it's a good practice to add a default value i think. I know the variable is defined to false in the files but it would break if it is ommited ```loadbalancer_apiserver_localhost|default(false)```